mkdocs-base.yml

```diff
@@ -121,185 +121,209 @@ extra:
+        - release-notes/Percona-Server-8.0.13-3.md
+        - release-notes/Percona-Server-8.0.12-2rc1.md
+  - Features:
+        - Audit Log Filter:
+          - audit-log-filter-overview.md
+          - install-audit-log-filter.md
+          - Formats: 
+            - Overview: audit-log-filter-formats.md
+            - XML (New style): audit-log-filter-new.md
+            - XML (Old style): audit-log-filter-old.md
+            - JSON: audit-log-filter-json.md
+          - audit-log-filter-security.md
+          - audit-log-filter-compression-encryption.md
+          - reading-audit-log-filter-files.md
+          - filter-audit-log-filter-files.md
+          - audit-log-filter-restrictions.md
+          - audit-log-filter-naming.md
+          - disable-audit-log-filter.md
+          - audit-log-filter-variables.md
+          - uninstall-audit-log-filter.md
+        - flexibility/binlog_space.md
+        - flexibility/extended_mysqlbinlog.md
+        - flexibility/slowlog_rotation.md
+        - flexibility/extended_select_into_outfile.md
+        - flexibility/proxy_protocol_support.md
+        - flexibility/sequence_table.md
+        - flexibility/trigger-updates.md 
+        - management/innodb_expanded_fast_index_creation.md
+        - management/kill_idle_trx.md
+        - management/udf_percona_toolkit.md
+        - management/utility_user.md
+        - performance/procfs-plugin.md
+        - performance/adaptive_network_buffers.md
+        - performance/threadpool.md
+  - Quickstart guide:
+       - quickstart-overview.md
+  - Install:
+      - Install Percona Server for MySQL from repositories: installation.md
       - Use APT:
-        - Install using APT repository: installation/apt-repo.md
+        - Use APT repositories: installation/apt-repo.md
         - Files in DEB package: installation/apt-files.md
         - installation/build-apt-packages.md
-        - Install from downloaded DEB packages: installation/apt-download-deb.md
+        - Downloaded DEB packages: installation/apt-download-deb.md
         - Apt pinning: installation/apt-pinning.md
         - Run Percona Server for MySQL: installation/apt-run.md
         - Uninstall: installation/apt-uninstall-server.md
       - Use YUM:
-        - Install using a RPM repository: installation/yum-repo.md
+        - Use RPM repositories: installation/yum-repo.md
         - Files in RPM package: installation/yum-files.md
-        - Install from downloaded RPM packages: installation/yum-download-rpm.md
+        - Downloaded RPM packages: installation/yum-download-rpm.md
         - Run Percona Server for MySQL: installation/yum-run.md
         - Uninstall: installation/yum-uninstall.md
       - Use binary tarballs:
         - Install with binary tarballs: installation/binary-tarball-install.md
         - Binary tarballs available: installation/binary-tarball-names.md
       - Compile from source:
         - installation/source-tarball.md
-        - installation/compile-percona-server.md
```

Comment:
Just make sure all these changes should be here, seem to be not related to the plugin